tgui/packages/tgui/interfaces/DropshipFlightControl.tsx

```diff
@@ -276,6 +278,61 @@ const AutopilotConfig = (props, context) => {
   );
 };
 
+const StopPlayLaunchAnnouncementAlarm = (_, context) => {
+  const { act } = useBackend<NavigationProps>(context);
+  return (
+    <Button
+      icon="ban"
+      onClick={() => {
+        act('stop_play_launch_announcement_alarm');
+      }}>
+      Stop Alarm
+    </Button>
+  );
+};
+
+const PlayLaunchAnnouncementAlarm = (_, context) => {
+  const { act } = useBackend<NavigationProps>(context);
+  return (
+    <Button
+      icon="rocket"
+      onClick={() => {
+        act('play_launch_announcement_alarm');
+      }}>
+      Start Alarm
+    </Button>
+  );
+};
+
+const LaunchAnnouncementAlarm = (_, context) => {
+  const { data, act } = useBackend<DropshipNavigationProps>(context);
+  const [siteselection, setSiteSelection] = useSharedState<string | undefined>(
+    context,
+    'target_site',
+    undefined
+  );
+  return (
+    <Section
+      title="Launch Announcement Alarm"
+      buttons={
+        <>
+          {!data.playing_launch_announcement_alarm && (
+            <PlayLaunchAnnouncementAlarm />
+          )}
+
+          {data.playing_launch_announcement_alarm && (
+            <Button
+              onClick={() => act('stop_play_launch_announcement_alarm')}
+              icon="triangle-exclamation">
+              Stop Launch Alarm
+            </Button>
+          )}
+        </>
+      }
+    />
```

Comment:
The contents of this section are empty because the only functionality you added is via buttons to the right of the section title. Consider adding them below (blue arrow). Or at least remove the empty space.

Also in this state there is a 0 (red arrow).

![image](https://github.com/cmss13-devs/cmss13/assets/76988376/cf1c7b51-a63d-4e1e-a4c0-9b4e211d43ab)

```suggestion
    <Section
      title="Launch Announcement Alarm"
      buttons={
        !data.playing_launch_announcement_alarm ? (
          <PlayLaunchAnnouncementAlarm />
        ) : (
          <StopLaunchAnnouncementAlarm />
        )
      }
    />
```

With above it would look like:

![image](https://github.com/cmss13-devs/cmss13/assets/76988376/beb0552f-2ba5-4ccb-9d79-e13a0b0010cb)

Reply:
I have no idea how to get rid of that zero and no idea where that bottom empty section comes from. 

I tried looking a bit through the "Section" file and didn't see anything. 

I also tried adding a </Section> to close the <Section ....... > tag and visual studio forces the code to become
<Section ........ />

There's nothing in the code that indicates a zero to be shown to my understanding. 

![image](https://github.com/cmss13-devs/cmss13/assets/142365554/ff542ecf-01bc-4053-b883-b3e4e0444a0a)
![image](https://github.com/cmss13-devs/cmss13/assets/142365554/aa2fa4a8-fdb2-4c40-ac99-aea610270f47)

Reply:
It was eliminated with my code suggestion, and has been eliminated in your latest commit c35cde6ce2a4312519ab51c2015b08891046ce8d